cli/src/lib.rs

```diff
@@ -79,13 +78,11 @@ pub struct Iroha {
     /// Kura — block storage
     pub kura: Arc<Kura>,
     /// Block synchronization actor
-    pub block_sync: AlwaysAddr<BlockSynchronizer>,
+    pub block_sync: Arc<BlockSynchronizer>,
```

Comment:
There's only one of those things, and it's pretty much static. Should we try and make it static?

Reply:
We should make them all static. After we have rewritten the tests so that they can support the change.

Reply:
Created #2919 resolving.